src/StarkEx.Crypto.SDK/Hashing/SpotTradingMessageHasher.cs

```diff
@@ -88,21 +111,79 @@ public BigInteger EncodeTransferWithFees(EncodeTransferWithFeesModel encodeTrans
         return pedersenHash.CreateHash(thirdInnerHash, fifthWeight);
     }
 
+    /// <inheritdoc />
+    public BigInteger EncodeTransferWithoutFees(EncodeTransferWithoutFeesModel encodeTransferWithoutFeesModel)
```

Comment:
This is functionally equivalent to `DeprecatedHashTransferOrder`.

Reply:
Deleted the existing deprecated method in favor of this one